static/styles/lib/bootstrap/scss/_dropdown.scss

```diff
@@ -53,12 +53,15 @@
   background-clip: padding-box;
   border: $dropdown-border-width solid $dropdown-border-color;
   @include border-radius($border-radius);
-  max-height: 400px;
-  overflow-y: auto;
```

Comment:
This is very global css changes, I'm not sure other dropdowns appear wrong. Maybe add your own class with !important settings?

Reply:
Those styles are not in the dropdown they are in other class 
.js-alert-content {
  max-height: 400px;
  overflow-y: auto;
}